### ISSUE TITLE
Blocks to break Black list

### ISSUE THREAD

**Issue by IXaar1** [AlexModGuy/Ice-and-Fire-Suggestions#272]
Dear developers, hello again.

We have a trouble with graves. Dragons always break them.
 That's why i want to suggest to add option(in config file) to disallow dragons breaking some custom blocks(idea is that i can add these block by myself)

I think it's possible to do.
Please answer :)
Goodbye for now,
Nikita

**Comment by mser7** [comment]
Are you using 1.5.2 version, and what grave mod did you used?

**Comment by IXaar1** [comment]
Yeah, version 1.5.2. And grave mod is Thomb many graves 2(i believe:) )

**Comment by mser7** [comment]
Strange, that glitch is supposed to be fixed in that version, I played around with it as well and the grave wasnt get destroyed
